packages/core/src/parsers/builder.ts

```diff
@@ -872,3 +879,20 @@ export const builderContentToMitosisComponent = (
 
   return componentJson;
 };
+function mapBuilderBindingsToMitosisBindingWithCode(
+  bindings: { [key: string]: string } | undefined,
+): MitosisNode['bindings'] {
+  const result: MitosisNode['bindings'] = {};
+  bindings &&
+    Object.keys(bindings).forEach((key) => {
+      const value: string | { code: string } = bindings[key] as any;
```

Comment:
Surprised that we needed this: I thought `BuilderElement` bindings will always be `bindings: { [key: string]: string }`, so how could the value be a `{ code: string }`? Did you encounter that possibility?

Reply:
If my understanding is correct, this is the only possibility here:

```suggestion
      const value: string = bindings[key]
```

Reply:
Yes the tests fail without this.